<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="paintworks_resource.qrc">
    <normaloff>:/images/images/paintworks.png</normaloff>:/images/images/paintworks.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>801</width>
      <height>561</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>824</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="statusTip">
     <string/>
    </property>
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_3D"/>
    <addaction name="actionSave"/>
    <addaction name="action_Q"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLine"/>
   <addaction name="actionCircle"/>
   <addaction name="actionEllipse"/>
   <addaction name="actionPolygon"/>
   <addaction name="actionCurve"/>
   <addaction name="separator"/>
   <addaction name="actionScalePlus"/>
   <addaction name="actionScaleMinus"/>
   <addaction name="separator"/>
   <addaction name="actionPickColor"/>
   <addaction name="actionFill"/>
   <addaction name="actionCut"/>
   <addaction name="separator"/>
   <addaction name="actionDelete"/>
   <addaction name="actionClear"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_New">
   <property name="text">
    <string>新建画布(&amp;N)</string>
   </property>
   <property name="toolTip">
    <string>新建画布</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Q">
   <property name="text">
    <string>退出(&amp;Q)</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="paintworks_resource.qrc">
     <normaloff>:/images/images/line.png</normaloff>:/images/images/line.png</iconset>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
   <property name="toolTip">
    <string>直线</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="paintworks_resource.qrc">
     <normaloff>:/images/images/circle.png</normaloff>:/images/images/circle.png</iconset>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
   <property name="toolTip">
    <string>圆</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/images/images/ellipse.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Ellipse</string>
   </property>
   <property name="toolTip">
    <string>椭圆</string>
   </property>
   <property name="shortcut">
    <string>Alt+E</string>
   </property>
  </action>
  <action name="actionPolygon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/images/images/polygon.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Polygon</string>
   </property>
   <property name="toolTip">
    <string>多边形</string>
   </property>
   <property name="shortcut">
    <string>Alt+P</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="paintworks_resource.qrc">
     <normaloff>:/images/images/cut.png</normaloff>:/images/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="toolTip">
    <string>裁剪</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionFill">
   <property name="icon">
    <iconset resource="paintworks_resource.qrc">
     <normaloff>:/images/images/fill.png</normaloff>:/images/images/fill.png</iconset>
   </property>
   <property name="text">
    <string>Fill</string>
   </property>
   <property name="toolTip">
    <string>填充</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionScalePlus">
   <property name="icon">
    <iconset resource="paintworks_resource.qrc">
     <normaloff>:/images/images/scaleplus.png</normaloff>:/images/images/scaleplus.png</iconset>
   </property>
   <property name="text">
    <string>ScalePlus</string>
   </property>
   <property name="toolTip">
    <string>放大</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionScaleMinus">
   <property name="icon">
    <iconset resource="paintworks_resource.qrc">
     <normaloff>:/images/images/scaleminus.png</normaloff>:/images/images/scaleminus.png</iconset>
   </property>
   <property name="text">
    <string>ScaleMinus</string>
   </property>
   <property name="toolTip">
    <string>缩小</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>另存为...</string>
   </property>
   <property name="toolTip">
    <string>保存文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_3D">
   <property name="text">
    <string>新建3D画布</string>
   </property>
   <property name="toolTip">
    <string>新建3D画布</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="paintworks_resource.qrc">
     <normaloff>:/images/images/delete.png</normaloff>:/images/images/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>删除</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="paintworks_resource.qrc">
     <normaloff>:/images/images/clear.png</normaloff>:/images/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>清空画布</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Del</string>
   </property>
  </action>
  <action name="actionPickColor">
   <property name="icon">
    <iconset resource="paintworks_resource.qrc">
     <normaloff>:/images/images/colorpicker.png</normaloff>:/images/images/colorpicker.png</iconset>
   </property>
   <property name="text">
    <string>pickColor</string>
   </property>
   <property name="toolTip">
    <string>选取颜色</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionCurve">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="paintworks_resource.qrc">
     <normaloff>:/images/images/curve.png</normaloff>:/images/images/curve.png</iconset>
   </property>
   <property name="text">
    <string>Curve</string>
   </property>
   <property name="toolTip">
    <string>曲线</string>
   </property>
   <property name="shortcut">
    <string>Alt+V</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="paintworks_resource.qrc"/>
 </resources>
 <connections/>
</ui>

